phpunit/abstracts/CommonITILRecurrentTest.php

```diff
@@ -521,16 +525,12 @@ public function testComputeNextCreationDate()
                 $calendars_id
             );
 
-            $this->assertSame($expected_value, $value);
+            $this->assertSame($expected_value, $value, 'Test case #' . $i);
```

Comment:
Since this isn't a real provider, we have to add the index ourselves to known which data from the "provider" failed.